smartsim/settings/batch_settings.py

```diff
@@ -117,6 +117,15 @@ def __init__(
             """The scheduler type"""
         except ValueError:
             raise ValueError(f"Invalid scheduler type: {batch_scheduler}") from None
+
+        if batch_args:
```

Comment:
Minor nit: Can we change this to
```suggestion
        if batch_args is not None:
```
just so that these do not errors do not get by the check?
```py
BatchSettings("slurm", batch_args="")
BatchSettings("slurm", batch_args=0)
BatchSettings("slurm", batch_args=[])
# etc.
```

Reply:
changed